src/ProjectNeon/Assets/Data/Cards/CardResolutionZone.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CardResolutionZone : ScriptableObject
+{
+    private List<PlayedCard> enemyMoves = new List<PlayedCard>();
```

Comment:
1) This isn't viewable in the Editor. Needs `SerializeField`
2) Why do we need to discriminate? Can't we put all PlayedCards together? 

If we need ordering logic, it can be handled by whichever object Resolves the cards.

Reply:
@SilasReinagel I've removed 2), and now we get all cards played in the same list.